srcpkgs/termrec/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'termrec'
+pkgname=termrec
+version=0.18
+revision=1
+build_style=gnu-configure
+makedepends="libcurl-devel"
+depends="libtermrec"
```

Comment:
Automatically found:

=> termrec-0.18_1: running pre-pkg hook: 04-generate-runtime-deps ...

   SONAME: libtty.so.1 <-> libtermrec-0.18_1

Reply:
I had it in mind there was some weird behavior with depending on sub-packages. Fixed.

Reply:
the depends line is unnecessary

Reply:
I could've sworn I removed that